inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/generic_eltwise_ref.cl

```diff
@@ -84,9 +87,16 @@ KERNEL(eltwise)(
 
     DO_ELTWISE;
 
+#if HAS_FUSED_OPS
+    FUSED_OPS;
+    OUTPUT_TYPE out = FUSED_OPS_RESULT;
+#else
+    OUTPUT_TYPE out = TO_OUTPUT_TYPE(res);
+#endif
+
 #if QUANTIZATION_TERM && !OUTPUT_IS_FP
-    output[output_offset] = TO_OUTPUT_TYPE_SAT(ACTIVATION(res, ACTIVATION_PARAMS));
+    output[output_offset] = TO_OUTPUT_TYPE_SAT(ACTIVATION(out, ACTIVATION_PARAMS));
```

Comment:
`ACTIVATION` or `ACTIVATION_TYPED` shouldn't be used together with FUSED_OPS, please move to `#else` of `#if HAS_FUSED_OPS`